### PR TITLE
Upgrade scala to 2.12.0 and sbt to 0.13.13

### PR DESCRIPTION
**Pull request by TimothyKlim**

### CODE REVIEW

**Comment by cvogt** [comment]
rebuilding.  maybe we need to upgrade scalatest to the final release? seems like it couldn't fine the RC we are using anymore

**Comment by TimothyKlim** [comment]
Yeah, https://github.com/scalatest/scalatest/issues/989

**Comment by cvogt** [comment]
ah appveyor passed, others on the way. Fingers crossed :)

**Comment by TimothyKlim** [comment]
Same errors. I'll wait for scalatest release (https://github.com/scalatest/scalatest/issues/989).

**Comment by szeiger** [comment]
I'll blame @viktorklang for this. If he didn't prefer TestNG over JUnit we wouldn't need any Scala-based dependencies :-P

**Comment by viktorklang** [comment]
@szeiger Hey, @sirthias can probably tell you a story about JUnit ;-)

**Comment by sirthias** [comment]
I will not ever speak the JU... word again, sorry @viktorklang!

File: project/Build.scala [hunk 1]
```diff
@@ -12,9 +12,9 @@ import com.typesafe.sbt.sdlc.Plugin._
 
 object SlickBuild extends Build {
 
-  val slickVersion = "3.2.0-SNAPSHOT"
+  val slickVersion = "3.2.0-M2"
```
Comment (szeiger) [inline project/Build.scala]: I'd prefer to keep SNAPSHOT here so you don't accidentally publish a non-snapshot version.

**Comment by dvic** [comment]
Any idea when this will be released to the central repo?

Thanks! 👍

**Pull request merged**